src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -32,6 +34,7 @@
 @CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
         oneLineSummary = "create a hadoop-bam splitting index" ,
         programGroup = SparkProgramGroup.class)
+@DocumentedFeature(groupName= HelpConstants.DOCS_CAT_DATA)
```

Comment:
You make a good point on dimensions, @cmnbroad. I wonder if perhaps the spark/non-spark nature can be captured in the walkertype? If so that would give us all we need to enable listing things based on arbitrary logic in webpages. E.g. we can have additional templates that organize tools differently and produce corresponding webpages; then we can present these organization options to users through clickable widgets.